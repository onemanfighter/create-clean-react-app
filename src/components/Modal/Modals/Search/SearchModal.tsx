import { Box } from '@chakra-ui/react';

const SearchModal = () => {
  return (
    <Box padding={1} top={100} zIndex={1000}>
      Search
    </Box>
  );
};

export default SearchModal;
