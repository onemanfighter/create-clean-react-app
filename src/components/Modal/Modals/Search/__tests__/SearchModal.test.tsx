import { render } from '@testing-library/react';
import SearchModal from '../SearchModal';

describe('Search modal', () => {
  it('Should render correctly', () => {
    const { container } = render(<SearchModal />);

    expect(container).toMatchSnapshot();
  });
});
