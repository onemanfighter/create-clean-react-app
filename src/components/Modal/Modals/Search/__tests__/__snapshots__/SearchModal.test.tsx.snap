// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search modal Should render correctly 1`] = `
<div>
  <div
    class="css-1n6jq51"
  >
    Search
  </div>
</div>
`;
